src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
must be a class field. No need to create a new random instance per each method invocation (as happens now)